server.crt

```diff
@@ -1,19 +0,0 @@
------BEGIN CERTIFICATE-----
```

Comment:
I know you rebased this on wip/tcp, but I think you could rebase on master instead and sqash this with the initial tls stuff that is already on wip/tcp, so that we never see the certifcates added at all